frontend/src/main/components/Users/UsersTable.js

```diff
@@ -16,16 +14,35 @@ export default function UsersTable({ users}) {
         }
     }
 
+    function cellToAxiosParamsToggleDriver(cell){
+        return {
+            url: "/api/admin/users/toggleDriver", // going to have to add api/driver
```

Comment:
What does this comment mean?  Do we still need it?

Reply:
No, forgot to remove it